llama_index/evaluation/base.py

```diff
@@ -25,9 +26,20 @@ class EvaluationResult(BaseModel):
     score: Optional[float] = Field(None, description="Score for the response")
 
 
-class BaseEvaluator(ABC):
+class BaseEvaluator(PromptMixin):
     """Base Evaluator class."""
 
+    # def _get_prompts(self) -> PromptDictType:
+    #     """Get prompts."""
+    #     return {}
+
+    # def _update_prompts(self, prompts: PromptDictType) -> None:
+    #     """Update prompts."""
```

Comment:
how come we don't have the default impl here? (but was there for agents)

Reply:
since most eval modules use prompts

wanted to avoid doing empty defaults unless most of the modules didn't use prompts